### PR TITLE
Update CODEOWNERS file

### PR DESCRIPTION
**Pull request by dsouzai**
* Remove Robert Young's github handle from CODEOWNERS
* Add Irwin D'Souza as codeowner for the control, env, runtime dirs

### CODE REVIEW

**Comment by dsouzai** [comment]
@0xdaryl could you review?

File: CODEOWNERS [hunk 1]
```diff
@@ -28,16 +28,19 @@
 *.yml @charliegracie
 
 # CMake
-*.cmake @youngar @charliegracie @rwy0717
-/cmake/ @youngar @charliegracie @rwy0717
-**/CMakeLists.txt @youngar @charliegracie @rwy0717
+*.cmake @youngar @charliegracie @rwy7
```
Comment (dsouzai) [inline CODEOWNERS:31]: For some reason it still complains about this handle, not sure why

Comment (0xdaryl) [reply CODEOWNERS:31]: Who is "it" ?

@rwy7 : Do you still want to be referenced in the CODEOWNERS file?  (hi, btw :-) )

Comment (dsouzai) [reply CODEOWNERS:31]: "it" being github; when you go to the "Files Changed" tab, there's an error message at the top.

Comment (hzongaro) [reply CODEOWNERS:31]: Is the problem that `rwy7` is not in the [list of committers](https://github.com/orgs/eclipse/teams/technology-omr-committers)?

```
Unknown owner on line 31: make sure @rwy7 exists and has write access to the repository 
```

Comment (dsouzai) [reply CODEOWNERS]: Made the requested changes.

**Review by 0xdaryl — APPROVED** [review]

**Pull request merged**